ledger_device_sdk/src/nbgl.rs

```diff
@@ -0,0 +1,267 @@
+use crate::io::{ApduHeader, Comm, Event, Reply};
+use const_zero::const_zero;
+use core::mem::transmute;
+use ledger_secure_sdk_sys::nbgl_icon_details_t;
+use ledger_secure_sdk_sys::*;
+
+pub struct NbglHome;
+pub struct NbglReview;
+pub struct NbglAddressConfirm;
+
+#[derive(PartialEq)]
+pub enum ReviewStatus {
+    Pending,
+    Validate,
+    Reject,
+}
+
+pub struct Field<'a> {
+    pub name: &'a str,
+    pub value: &'a str,
+}
+
+struct NbglContext {
```

Comment:
This struct should be cleaned-up, most of it, if not all of it probably is not needed with the nbgl sync API and could be replaced by fields of `NbglHome`, `NbglReview` and `NbglAddressConfirm`

Reply:
Done.